dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/DocumentControllerTest.kt

```diff
@@ -91,13 +95,63 @@ class DocumentControllerTest {
         assertEquals(HttpStatus.NOT_FOUND.value(), exception.statusCode)
     }
 
+    @Test
+    fun `test that non admin users can upload documents only when they are CompanyOwner and not otherwise`() {
```

Comment:
Würde die Testbeschreibung leicht anpassen, weil sie nicht ganz der Wahrheit entspricht:
Denn User mit keycloak-uploader Rolle sind ja auch non-admins, können aber trotzdem Dokumente hochladen.

Vorschlag für die Änderung:

`test that users without keycloak-uploader-role can upload documents only if assigned company-owner-role`